From 80bd32b7da1f70b16af815d61ef71db20c564434 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:34 -0400
Subject: [redhat] kabi: Adding symbol param_set_uint (kernel/params.c)

Message-id: <83568b47d1571a9dfcc6ad83d282331b110ea478.1410510773.git.poros@redhat.com>
Patchwork-id: 96000
O-Subject: [PATCH RHEL7.1 154/233] kabi: Adding symbol param_set_uint (kernel/params.c)
Bugzilla: 1071099
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Not used directly, used by module_param internally for type uint.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_set_uint
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_set_uint
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_set_uint

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_set_uint b/redhat/kabi/kabi-module/kabi_ppc64/param_set_uint
new file mode 100644
index 0000000..eb5a2eb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_set_uint
@@ -0,0 +1,2 @@
+#1-
+0x8bfe8c57 param_set_uint vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_set_uint b/redhat/kabi/kabi-module/kabi_s390x/param_set_uint
new file mode 100644
index 0000000..eb5a2eb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_set_uint
@@ -0,0 +1,2 @@
+#1-
+0x8bfe8c57 param_set_uint vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_set_uint b/redhat/kabi/kabi-module/kabi_x86_64/param_set_uint
new file mode 100644
index 0000000..eb5a2eb
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_set_uint
@@ -0,0 +1,2 @@
+#1-
+0x8bfe8c57 param_set_uint vmlinux EXPORT_SYMBOL
-- 
1.7.1